src/Dialog/confirmation-dialog.scss

```diff
@@ -0,0 +1,10 @@
+@import "../variables";
+
+.modal {
+    background-color: $modal-backdrop-bg;
+}
+
+.modal-content {
+    border-radius: 0px !important;
```

Comment:
This (like the whole file) will effect not just confirmation dialogs but _every_ modal. Is this your intention? 

If yes, then put it in a more central place to indicate this (currently, I suggest `shared.scss` for it), if not, then add a specific class name to the modal of the confirmation dialog and also use it here (which would also make it unnecessary to use `!important`, since it would increase the specificity of the selector).

Reply:
Yes, it is my intention, I will move it to shared.scss.